makei/cvtsrcpf.py

```diff
@@ -0,0 +1,114 @@
+#!/QOpenSys/pkgs/bin/python3.6
+# -*- coding: utf-8 -*-
+import argparse
+from fileinput import filename
+import os
+from pathlib import Path
+import re
+import shutil
+import sys
+from typing import Any, Dict, List, Optional, Tuple
+from unittest import result
+
+sys.path.append(str(Path(__file__).resolve().parent.parent))  # nopep8
+from makei.utils import Colors, colored, create_ibmi_json, objlib_to_path, validate_ccsid
+from makei.ibm_job import IBMJob  # nopep8
+
+
+class CvtSrcPf():
+    """convert from source physical file
+    """
+    job: IBMJob
+
+    lib: str
+    srcfile: str
+    save_path: Path
+    stmf_ccsid: str
+    tgt_ccsid: str
+    ibmi_json_path: Optional[Path]
+
+
+    def __init__(self, srcfile: str, lib: str, stmf_ccsid: str = "1208", should_create_ibmi_json: bool = True, tgtccsid: str = "*JOB", save_path: Path = Path.cwd()) -> None:
+        self.job = IBMJob()
+
+        self.lib = lib
+        self.srcfile = srcfile
+        self.save_path = save_path
+        if validate_ccsid(stmf_ccsid):
+            self.stmf_ccsid = stmf_ccsid
+        else:
+            raise Exception(f"Invalid ccsid {stmf_ccsid}")
+        if validate_ccsid(tgtccsid):
```

Comment:
use the src-pf CCSID as the .ibmi.json CCSID
unless 65535 - then use parameter
if not then use job CCSID